src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -1264,7 +1261,9 @@ bool QMCFixedSampleLinearOptimize::one_shift_run()
   // build the overlap and hamiltonian matrices
   optTarget->fillOverlapHamiltonianMatrices(hamMat, ovlMat);
   invMat.copy(ovlMat);
-
+  
+  // DEBUG: This is supposed to print the
```

Comment:
It should be activated with `<parameter name="output_matrices_csv">yes</parameter>` in the optimizer input.  The output goes to files with `.ovl` and '.ham' in the names.

Reply:
Fixed.